plugins/tiddlywiki/multiwikiserver/modules/routes/handlers/get-anon.js

```diff
@@ -0,0 +1,44 @@
+/*\
+title: $:/plugins/tiddlywiki/multiwikiserver/routes/handlers/get-anon.js
+type: application/javascript
+module-type: mws-route
+
+GET /admin/anon
+
+\*/
+(function() {
+
+/*jslint node: true, browser: true */
+/*global $tw: false */
+"use strict";
+
+exports.method = "GET";
+
+exports.path = /^\/admin\/anon\/?$/;
+
+exports.handler = function(request, response, state) {
+  // Check if user is authenticated and is admin
+  if(!state.authenticatedUser || !state.authenticatedUser.isAdmin) {
+    response.writeHead(401, "Unauthorized", { "Content-Type": "text/plain" });
+    response.end("Unauthorized");
+    return;
+  }
+
+  
+  // Update the configuration tiddlers
+  var wiki = $tw.wiki;
+  wiki.addTiddler({
+    title: "$:/config/MultiWikiServer/AllowAnonymousReads",
+    text: "undefined"
+  });
+  wiki.addTiddler({
+    title: "$:/config/MultiWikiServer/AllowAnonymousWrites",
+    text: "undefined"
+  });
```

Comment:
Why does a GET command create config tiddlers? The state should be initialized in a "safe" way by default. 
See my other comment. Be **deleting** the config tiddlers, the system should be in a "safe" state.